### PR TITLE
Add: macos aarch64 release

### PR DESCRIPTION
**Pull request by 21st-centuryman**
I wanted to add this package to homebrew so this would be the first step in achieving this.

Love the package btw. cheers

### CODE REVIEW

**Comment by 21st-centuryman** [comment]
Also if there is anything I can add/contribute to with this just hmu.

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -69,7 +70,16 @@ jobs:
             target: x86_64-apple-darwin
             artifact_name: target/x86_64-apple-darwin/release/pfetch
             release_name: pfetch-macos-x86_64
-            cross: false
+            cross: true
```
Comment (Gobidev) [inline .github/workflows/release.yml:73]: `cross` is not needed for the x86 build, as the job runs on macos x86_64

**Review by Gobidev — CHANGES_REQUESTED** [review]
Thank you for offering your help^^
Currently these are the things I have planned for this project:
- add Windows binary to release workflow that is not .tar.gz compressed

- make the following changes in `libmacchina`:
   - add Wayland WM detection
   - find fast method to count `nix` packages
   - count Windows `winget` and `chocolatey` packages

Aside from that, the project is more or less done, as it is supposed to be minimalistic. So unless someone has an awesome idea, there aren't really any big features to work on.

**Comment by Gobidev** [comment]
I added a macos aarch64 binary to the latest release manually, so you should be able to add it to homebrew if you want :)

**Comment by 21st-centuryman** [comment]
Awesome. I will set this up when I find the time. 

Thanks for your help!

**Comment by 21st-centuryman** [comment]
PR created:
https://github.com/Homebrew/homebrew-core/pull/125251

Bear in mind it is my first homebrew formula PR so I might have overlooked something.

**Comment by Gobidev** [comment]
All right, thank you!

**Pull request merged**